.github/workflows/healthchecks_azurekeyvault_secrets_cd.yml

```
@@ -0,0 +1,29 @@
+name: HealthChecks AzureKeyVault Secrets CD
+
+on:
+  push:
+    tags:
+      - release-azurekeyvault-secrets-*
+      - release-all-*
+
+jobs:
+  build:
+    env:
+      BUILD_CONFIG: Release
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: |
+            6.0.x
+            7.0.x
+      - name: Restore
+        run: dotnet restore ./src/HealthChecks.AzureKeyVault/HealthChecks.AzureKeyVault.Secrets.csproj
+      - name: Build
+        run: dotnet build --no-restore ./src/HealthChecks.AzureKeyVault.Secrets/HealthChecks.AzureKeyVault.Secrets.csproj -c $BUILD_CONFIG
+      - name: Pack
+        run: dotnet pack --no-build ./src/HealthChecks.AzureKeyVault.Secrets/HealthChecks.AzureKeyVault.Secrets.csproj -c $BUILD_CONFIG -o ./artifacts
+      - name: Publish
+        run: dotnet nuget push ./artifacts/AspNetCore.HealthChecks.AzureKeyVault.Secrets.*.nupkg -k ${{secrets.NUGET_API_KEY}} -s https://api.nuget.org/v3/index.json --skip-duplicate
```

the paths miss a dot in the name: HealthChecks.Azure**.**KeyVault.Secrets

```suggestion
        run: dotnet restore ./src/HealthChecks.Azure.KeyVault.Secrets/HealthChecks.Azure.KeyVault.Secrets.csproj
      - name: Build
        run: dotnet build --no-restore ./src/HealthChecks.Azure.KeyVault.Secrets/HealthChecks.Azure.KeyVault.Secrets.csproj -c $BUILD_CONFIG
      - name: Pack
        run: dotnet pack --no-build ./src/HealthChecks.Azure.KeyVault.Secrets/HealthChecks.Azure.KeyVault.Secrets.csproj -c $BUILD_CONFIG -o ./artifacts
      - name: Publish
        run: dotnet nuget push ./artifacts/AspNetCore.HealthChecks.Azure.KeyVault.Secrets.*.nupkg -k ${{secrets.NUGET_API_KEY}} -s https://api.nuget.org/v3/index.json --skip-duplicate
```